rust/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "rust-ws-server"
+version = "0.1.0"
+authors = [""]
+
+[dependencies]
+ws = "0.5.1"
```

Comment:
It would be really cool to see a benchmark for this built on top of tokio, though it's still early days. A second implementation to compare, maybe :)